src/assets/TokenListController.test.ts

```diff
@@ -625,11 +873,9 @@ describe('TokenListController', () => {
     const messenger = getRestrictedMessenger();
     const controller = new TokenListController({
       chainId: NetworksChainId.mainnet,
-      useStaticTokenList: false,
       onNetworkStateChange: (listener) => network.subscribe(listener),
-      onPreferencesStateChange: (listener) => preferences.subscribe(listener),
       messenger,
-      state: expiredCacheExistingState,
+      state: expiredCacheExistingState as any,
```

Comment:
Is it possible to not typecast this?

Reply:
Removed this `any` and typed the object to be `TokenListState` from `TokenListController` instead.